resources/buildConfigDefinitions.js

```diff
@@ -0,0 +1,221 @@
+function last(array) {
```

Comment:
a comment for this file would be super helpful.  I'm reasonably familiar with the code base but could def use some clues as to what is going on here.....

Reply:
Yep for sure!